lsp-metals.el

```diff
@@ -365,13 +365,55 @@ change it again."
       (let ((stacktrace (buffer-substring (region-beginning) (region-end))))
         (lsp-send-execute-command "metals.analyze-stacktrace" (vector stacktrace))))))
 
+
```

Comment:
Any reason to add this new line? :)

Reply:
None! I'm definitely not experienced with emacs lisp style or usage :) Any advice is appreciated.